src/Dijkstra/AST/Branching.agda

```diff
@@ -279,6 +269,9 @@ module BranchingSyntax (BaseOps : ASTOps) where
                                   ; (lift false) → e
                                   }
 
```

Comment:
I would rewrite this comment as:
```
'maybeAST' follows the same argument order 'maybe' in Agda's stdlib.
```

Do not mention "flip it below" here - since does "below" mean somewhere in the "maybeAST" or somewhere else.  Besides, it is redundant - the MaybeD instance says what it does.

A rule of comments is "don't repeat what the code does".

You might put in WHY it is flipped (not the fact that it is flipped).  But if you do, put it next to where flip is used, not in the description of "maybeAST"

Reply:
DONE